src/hooks/useQueryParams.ts

```diff
@@ -1,15 +0,0 @@
-import React from 'react';
-
-import { useLocation } from 'react-router-dom';
-
-export function useQuery() {
-  const { search } = useLocation();
-
-  return React.useMemo(() => new URLSearchParams(search), [search]);
-}
```

Comment:
It's worth mentioning that this was in React Router already :) https://reactrouter.com/en/main/hooks/use-search-params